tests/hpc/spack_master.pm

```diff
@@ -20,23 +20,12 @@ sub run ($self) {
     my $mpi_bin = 'mpi_bin';
     my @cluster_nodes = $self->cluster_names();
     my $cluster_nodes = join(',', @cluster_nodes);
+    my %exports_path = (bin => '/home/bernhard/bin');
```

Comment:
can't we just do my $exports_path = '/home/bernhard/bin'; ? 
I'm not sure but I don't see why not :-)

Reply:
The only reason is because i want to keep the loop and add directory on request. If i change that i will have to touch other parts too. and when/if something else need to be shared it is gonna be back and forth. As it it is, you need only to add/remove paths in the list